package.json

```diff
@@ -17,18 +17,30 @@
     "url": "git://github.com/helmetjs/content-security-policy-parser.git"
   },
   "bugs": "https://github.com/helmetjs/content-security-policy-parser/issues",
+  "main": "./dist/index.js",
+  "typings": "./dist/index.d.ts",
+  "files": [
+    "LICENSE",
+    "README.md",
+    "dist/index.js",
+    "dist/index.d.ts"
+  ],
   "scripts": {
-    "pretest": "standard --fix",
```

Comment:
Because `standard` is being removed, we should remove its badge from the readme.